images/README.md

```diff
@@ -1,18 +1,14 @@
 # Dev Images
 
-This folder contains a docker-compose file for development. To create and start the service execute `docker-compose up -d`.
-This will produce 2 following images:
+This folder contains a docker-compose file for development. It creates and starts the service which has two containers.
 
-* `beeyond/keycloak-dev`
-* `beeyond/postgres-dev`
+## PostgreSQL
 
-## PostgreSQL (postgres-dev)
+The container `beeyond-database` specifies the PostgreSQL **database** `beeyond` which has an **user** `beeyond` with the **password** `beeyond`.
```

Comment:
'The container `beeyond-database`' -> 'The Postgres container `beeyond-database`'.

database is `beeyond_db` and not `beeyond`